packages/protobuf/src/wire/binary-encoding.ts

```diff
@@ -398,10 +400,15 @@ export class BinaryReader {
         this.pos += len;
         break;
       case WireType.StartGroup:
-        // TODO check for matching field numbers in StartGroup / EndGroup tags
-        let t: WireType;
-        while ((t = this.tag()[1]) !== WireType.EndGroup) {
-          this.skip(t);
+        for (;;) {
```

Comment:
Interesting syntax. Is this idiomatic for JS/TS? In languages with `while` keywords, I am used to instead seeing `while (true)`.

Reply:
It would be idiomatic to avoid loops, which is not feasible here. I would prefer `while (true)` as well, but the linter objects the constant condition. 

It's possible that the linter can be configured to accept it, but touching the linter config is always very involved. Since we have to change the linter config to replace `make` with `turborepo`, it's better to punt on it for now. Our usage of `make` in this repo is much more unidiomatic than this loop construct.